### PR TITLE
[SYTO-978] Video Modal CSS stuff

### PR DESCRIPTION
**Pull request by sebasrgl**
https://jira.natgeo.com/browse/SYTO-978

#### How to test
1. ``gulp serve`` and ``gulp test``
2. Go to components and scroll down until the video promo component and then click it.
3. Check the AC is met.

### CODE REVIEW

File: src/modules/videomodal/_styles.scss [hunk 1]
```diff
@@ -52,6 +52,10 @@ $mt3_color-inverse: $mt3_color-white;
 
   &__title {
     font-size: $mt3_h-med9 + (2 / 16);
+
+    @media (max-width: 767px) {
```
Comment (pmourelle) [inline src/modules/videomodal/_styles.scss]: Use a variable for this :)

File: src/modules/videomodal/VideoModal.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class VideoModal extends Component {
     this.duration = data.text.duration ? data.text.duration : null;
     this.title = data.text.title ? data.text.title : null;
     this.dek = data.text.dek ? data.text.dek : null;
-    
+    VideoModal.toggleBodyOverflow();
```
Comment (pmourelle) [inline src/modules/videomodal/VideoModal.js]: If your intention is to have this class set, you may ensure it passing a `true` as parameter and pass it (in toggleBodyOverflow) to `classList.toggle`.
```
element.classList.toggle("classToAdd", true);
```

**Comment by krlospelaez** [comment]
PR Ok!

**Comment by pmourelle** [comment]
Looks good!

**Pull request merged**